src/Microsoft.DotNet.Cli.Utils/project.json

```diff
@@ -15,14 +15,7 @@
     "NuGet.ProjectModel": "3.6.0-rc-1954"
   },
   "frameworks": {
-    "net451": {
-      "frameworkAssemblies": {
-        "System.Runtime": {
-          "type": "build"
-        }
-      }
-    },
-    "netstandard1.6": {
+    "netstandard1.3": {
```

Comment:
I think we agreed on this going to netstandard1.5. Can you make the change here?